### DISCUSSION TITLE
Is there a way to install scoop for all users?

### DISCUSSION THREAD

**Discussion by IveMalfunctioned** [ScoopInstaller/Scoop#4658] [Help and Questions] [answered]
Title
Please do not link issue #3875 here, it does not work

**Comment by rashil2000** [comment, marked as answer]
What have you tried so far? Where are you facing problems? You provided zero information. How do you expect to get help like this.

**Comment by IveMalfunctioned** [comment]
To answer your questions:
1. Issue #3875, the one you keep shoving in my face
2. Issue #3875 does not work
3. By having people replying with help.

**Reply by rashil2000** [reply]
> 1. Issue [Is it possible to install Scoop itself globally, for all users? #3875](https://github.com/ScoopInstaller/Scoop/issues/3875), the one you keep shoving in my face

You say this, and then

> 3. By having people replying with help.

Expect this. I suppose this is your first time interacting with any forum online - there's no other explanation for this sense of entitlement.

Did you read this comment in the linked issue https://github.com/ScoopInstaller/Scoop/issues/3875#issuecomment-1007898847? Good chance you didn't.

Again, "does not work" is useless information. You need to tell the exact errors/steps/problems that you face with the workarounds listed above. A million things could go wrong, leading to "does not work". No one here has time to guess what's happening with you. Be precise.

**Comment by IveMalfunctioned** [comment]
Dude, linking a 2 year old unsupported solution and continuously telling me it's what I need isn't going to get you any different results. Why the hell are you being so passive aggressive? Yes, I expect help because this is a help forum. Why would I post if I was not expecting help? I am not being entitled in any way, you're just being a dickhead. "does not work" is perfectly usable information, and that's all you need to know. Either provide something that will actually contribute to this thread or get out.
